gadget/ondisk.go

```diff
@@ -195,7 +201,7 @@ func (dl *DeviceLayout) CreateMissing(pv *gadget.LaidOutVolume) ([]DeviceStructu
 
 // RemoveCreated removes partitions added during a previous failed install
 // attempt.
-func (dl *DeviceLayout) RemoveCreated() error {
+func (dl *OnDiskVolume) RemoveCreated() error {
```

Comment:
same, not a blocker but it feels this belong as a function to gadget/install

Reply:
Added TODO.